packages/create-web/index.js

```diff
@@ -0,0 +1,215 @@
+import path from "node:path";
+import fs from "node:fs";
+import { fileURLToPath } from 'node:url';
+
+export function setupTemplate(
+    dstDir, // string
+    projectName, // string
+    backendLanguage, // "python" | "rust" | "none"
+) {
+    const pathToTemplate = path.join(
+        path.dirname(fileURLToPath(import.meta.url)),
+        "template"
+    );
+
+    // TODO: Don't copy things that match gitignore
+    // Copy files that will always be there
+    cp_r(".gitignore", pathToTemplate, dstDir);
```

Comment:
gitignore file is not present on npm package

Reply:
Resolved, see code in later commits, and also https://stackoverflow.com/a/78712387/26116613 for explanation